skyVegasTests/utilities/index.ts

```diff
@@ -0,0 +1,3 @@
+export { launchGame } from './launchGameHelper';
+export { validateConsoleMessages } from './validateConsoleMessagesHelper';
+export { login } from './loginHelper';
```

Comment:
Any reason for consolidating all in one file?

Reply:
I read it was good to simplify the import line at the top of the file. Since they are all in different locations now I found this as a way of doing it. Not sure if it is the best way so open to suggestions